### PR TITLE
i18n(es): Update of `rss.md`, `styling.md`,  `typescript.md` & `manual.md`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated content of different markdown files, listed as follows:
-`rss.md` based on this commit 039d95ca28c60c799006f3b7a10510aae096dd4c
- `styling.md` based on these commits 314ba692b76885bf93e72d9553cd82317526831f, a5386d1c683ac9b4ce161e4ae955dec3e7805f3e & 578bed045f87968005ba56462f1bcfa75a173a80
- `typescript.md` based on this commit af0e63e8aa7e39bdb26aa9021dc3af3c9473d1ec
- `manual.md` based on these commits e96c0ca152cf93f49ed46e2be16c6867a61781cb & 79bdbb9ff909981606a62e276922a16bcd74e47c

### CODE REVIEW

File: src/pages/es/guides/styling.md [hunk 1]
```diff
@@ -112,14 +131,19 @@ const { class: className } = Astro.props;
 ---
 import MyComponent from "../components/MyComponent.astro"
 ---
-<style is:global>
+<styl>
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/styling.md:134]:
```suggestion
<style>
```

File: src/pages/es/guides/styling.md [hunk 2]
```diff
@@ -112,14 +131,19 @@ const { class: className } = Astro.props;
...
     color: red;
   }
 </style>
 <MyComponent class="red">¡Esto será rojo!</MyComponent>
 ```
 
+Este patrón te permite estilar componentes hijos directamente. Astro pasará el nombre de la clase local del padre (por ejemplo `astro-HHNQFKH6`) por la propiedad `class` automaticamente, incluyendo el hijo en ámbito del componente padre.
+
+:::note[Clases globales de componentes padre]
+Debido a que la propiedad `clase` incluye al hijo en el ámbito de su padre, es posible que los estilos caigan en cascada de padre a hijo. Para evitar que esto tenga efectos secundarios no deseados, asegúrese de usar nombres de clase únicos en el componente secundario.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/styling.md:145]:
```suggestion
Debido a que la propiedad `class` incluye al hijo en el ámbito de su padre, es posible que los estilos caigan en cascada de padre a hijo. Para evitar que esto tenga efectos secundarios no deseados, asegúrese de usar nombres de clase únicos en el componente secundario.
```

**Review by kevinzunigacuellar — COMMENTED** [review]
gracias por la PR @dreyfus92 🚀

**Comment by dreyfus92** [comment]
Muchas gracias por el review @kevinzunigacuellar 💚

**Pull request merged**